.circleci/config.yml

```diff
@@ -146,17 +153,20 @@ workflows:
   version: 2
   build_and_test:
     jobs:
-      - build
       - wait_openfisca
+      - install
       - lint:
           requires:
-            - build
-      - test_mocha:
+            - install
+      - build:
           requires:
             - lint
```

Comment:
Why build requires lint ?

Reply:
Because there is no point in building if the files are not linted 🙂